### PR TITLE
 Update Documentation

### PR DESCRIPTION
**Pull request by ghost**
Update the documentation of ploneorg.addonlisting covering the following sections:
- Aim of the project
- implementation aspects 
     - model fields - what do the fields and fieldset represent

addresses #6

### CODE REVIEW

File: docs/implementation.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Implementation of ploneorg.addonlisting 
+===============================================================
+
+Plone Dexterity Content type is used to store the add-on data locally.
+
+The custom content type addOn consists of the following fields:
+
+* title – Name of the add-on and it is a mandatory field.
+* description - A one-line description for the add-on (up to 120 characters); if not specified,
```
Comment (loechel) [inline docs/implementation.rst:9]: Documentation is not code, line length might not fit for doc texts. 
Please read and apply https://docs.plone.org/about/rst-styleguide.html 

Line Break at each "." and not a line break within a sentence.

File: docs/implementation.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* docs_link – Link to this add-on's documentation
+* package_url – Link to this addon's pypi page
+* release_url – Link to this addon's release page
+* home_page – Link to this addon's home page
+* platform – List of compatible platforms
+* license - License assosciated with the addon
+* downloads - Number of downloads of this addon. It is used to sort the addons based on their download numbers.
+* upload_time - upload time of the first version of this addon to find whether it is a new addon. It is used to
+  find the newest addons to be displayed. 
+* cheesecake_code_kwalitee_id – It refers to the code kwalitee index of the package computed by the cheeescake script.
```
Comment (loechel) [inline docs/implementation.rst:45]: Nice catch, I haven't seen that those fields are also presented, Do you have a documentation link for those fields?

**Review by loechel — APPROVED** [review]
Good Start

File: docs/implementation.rst (hunk 2 above)
Comment (ghost) [reply docs/implementation.rst:45]: I referred to a sample package like https://pypi.python.org/pypi/wincertstore/json for the field names and also this blog  https://martin-thoma.com/analyzing-pypi-metadata/

File: docs/implementation.rst (hunk 1 above)
Comment (ghost) [reply docs/implementation.rst:9]: I'll keep that in mind and edit it

**Comment by tkimnguyen** [comment]
@pavithirakc do you want to add any more commits to this PR before we/I merge it?

**Comment by ghost** [comment]
no more commits to this PR @tkimnguyen

**Pull request merged**